app/src/main/java/com/example/android/lifecycles/step5_solution/Fragment_step5.java

```diff
@@ -47,7 +52,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         View root = inflater.inflate(R.layout.fragment_step5, container, false);
         mSeekBar = root.findViewById(R.id.seekBar);
 
-        mSeekBarViewModel = ViewModelProviders.of(getActivity()).get(SeekBarViewModel.class);
+        mSeekBarViewModel = new ViewModelProvider(requireNonNull(getActivity())).get(SeekBarViewModel.class);
```

Comment:
Can use `requireActivity()` instead of `requireNonNull(...)`

Reply:
done